### PR TITLE
Glossary - remove explicit summary in preparation for markdowning

### PR DESCRIPTION
**Pull request by hamishwillee**
Also fixes some ids and classes that just aren't needed

### CODE REVIEW

File: files/en-us/glossary/node/dom/index.html [hunk 1]
```diff
@@ -5,7 +5,9 @@
   - CodingScripting
   - Glossary
 ---
-<p><span class="seoSummary">In the context of the {{Glossary("DOM")}}, a <strong>node</strong> is a single point in the node tree. Various things that are nodes are the document itself, elements, text, and comments.</span></p>
+<p>In the context of the {{Glossary("DOM")}}, a <strong>node</strong> is a single point in the node tree. Various things that are nodes are the document itself, elements, text, and comments.</p>
+
+<p></p>
```
Comment (teoli2003) [inline files/en-us/glossary/node/dom/index.html:10]: This is weird.

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/glossary/node/dom/index.html (hunk 1 above)
Comment (hamishwillee) [reply files/en-us/glossary/node/dom/index.html]: Thanks. Ran my regexp in the wrong order. Fixed now.

**Review by teoli2003 — APPROVED** [review]

File: files/en-us/glossary/user_agent/index.html [hunk 2]
```diff
@@ -12,15 +12,15 @@
...
 
-<p><span class="objectBox objectBox-string">Spam bots, download managers, and some browsers often send a fake UA string to announce themselves as a different client. This is known as <em>user agent spoofing</em>.</span></p>
+<p>Spam bots, download managers, and some browsers often send a fake UA string to announce themselves as a different client. This is known as <em>user agent spoofing</em>.</p>
 
-<p><span class="objectBox objectBox-string">The user agent string can be accessed with </span>{{Glossary("JavaScript")}}<span class="objectBox objectBox-string"> on the client side using the {{domxref("Navigator/userAgent", "NavigatorID.userAgent")}} property.</span></p>
+<p>The user agent string can be accessed with {{Glossary("JavaScript")} on the client side using the {{domxref("Navigator/userAgent", "NavigatorID.userAgent")}} property.</p>
```
Comment (teoli2003) [inline files/en-us/glossary/user_agent/index.html:21]:
```suggestion
<p>The user agent string can be accessed with {{Glossary("JavaScript")}} on the client side using the {{domxref("Navigator/userAgent", "NavigatorID.userAgent")}} property.</p>
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
One typo in macro.

**Review by teoli2003 — APPROVED** [review]

File: files/en-us/glossary/user_agent/index.html (hunk 2 above)
Comment (hamishwillee) [reply files/en-us/glossary/user_agent/index.html]: Good catch. Thank you!

**Pull request merged**